### PR TITLE
New version: nkdAgility.AzureDevOpsMigrationTools.Preview version 15.1.2-Preview.3

### PR DESCRIPTION
**Pull request by MrHinsh**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162853)

### CODE REVIEW

File: manifests/n/nkdAgility/AzureDevOpsMigrationTools/Preview/15.1.2-Preview.3/nkdAgility.AzureDevOpsMigrationTools.Preview.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: nkdAgility.AzureDevOpsMigrationTools.Preview
+PackageVersion: 15.1.2-Preview.3
+PackageLocale: en-US
+Publisher: naked Agility with Martin Hinshelwood & Co.
+PackageName: Azure DevOps Migration Tools (Preview)
+License: MIT Licence
```
Comment (Trenly) [inline manifests/n/nkdAgility/AzureDevOpsMigrationTools/Preview/15.1.2-Preview.3/nkdAgility.AzureDevOpsMigrationTools.Preview.locale.en-US.yaml:9]:
```suggestion
License: MIT License
```

File: manifests/n/nkdAgility/AzureDevOpsMigrationTools/Preview/15.1.2-Preview.3/nkdAgility.AzureDevOpsMigrationTools.Preview.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: nkdAgility.AzureDevOpsMigrationTools.Preview
+PackageVersion: 15.1.2-Preview.3
+PackageLocale: en-US
+Publisher: naked Agility with Martin Hinshelwood & Co.
+PackageName: Azure DevOps Migration Tools (Preview)
+License: MIT Licence
+ShortDescription: Azure DevOps Migration Tools allow you to migrate Teams, Backlogs, Tasks, Test Cases, and Plans & Suits from one Project to another in Azure DevOps / TFS both within the same Organisation, and between Organisations.
```
Comment (Trenly) [inline manifests/n/nkdAgility/AzureDevOpsMigrationTools/Preview/15.1.2-Preview.3/nkdAgility.AzureDevOpsMigrationTools.Preview.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Azure DevOps Migration Tools allow you to migrate Teams, Backlogs, Tasks, Test Cases, and Plans & Suits from one Project to another in Azure DevOps / TFS both within the same Organization, and between Organizations.
```

**Review by Trenly — CHANGES_REQUESTED** [review]
```
Command line alias added: "devopsmigration"
Path environment variable modified; restart your shell to use the new value.
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName                            DisplayVersion   Publisher                                   ProductCode
-----------                            --------------   ---------                                   -----------
Azure DevOps Migration Tools (Preview) 15.1.2-Preview.3 naked Agility with Martin Hinshelwood & Co. nkdAgility.AzureDevOpsMigrationTools.Preview__...


PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> devopsmigration

Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Extensions.Hosting.Abstractions, Version=8.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' or one of its dependencies. The system cannot find the file specified.
   at VstsSyncMigrator.ConsoleApp.Program.<Main>d__0.MoveNext()
   at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine)
   at VstsSyncMigrator.ConsoleApp.Program.Main(String[] args)
   at VstsSyncMigrator.ConsoleApp.Program.<Main>(String[] args)
   ```
@MrHinsh - When I attempt to run this program, I get a file not found error. Is there a dependency which is missing?

**Comment by MrHinsh** [comment]
I have tested and validated the package. All dependencies are accounted for and everything runs as normal.

**Pull request closed without merging**